src/datumaro/plugins/validators.py

```diff
@@ -547,11 +547,31 @@ def compute_statistics(self, dataset):
 
         stats, filtered_anns = self._compute_common_statistics(dataset)
 
+        label_cat = dataset.categories()[AnnotationType.label]
+        label_groups = label_cat.label_groups
+
+        label_name_to_group = {}
+        for label_group in label_groups:
+            for idx, label_name in enumerate(label_group.labels):
+                if label_group.group_type == 0:
+                    label_name_to_group[label_name] = label_group.name
+                else:
+                    label_name_to_group[label_name] = label_group.name + f"_{idx}"
+
+        undefined_label_name = list(stats["label_distribution"]["undefined_labels"].keys())
+
         stats["items_with_multiple_labels"] = []
         for item_key, anns in filtered_anns:
-            ann_count = len(anns)
-            if ann_count > 1:
-                stats["items_with_multiple_labels"].append(item_key)
+            occupied_groups = set()
+            for ann in anns:
+                if ann.label in undefined_label_name:
+                    continue
+                label_name = label_cat[ann.label].name
+                label_group = label_name_to_group.get(label_name, "")
+                if label_group in occupied_groups:
```

Comment:
For better interpretability,
```suggestion
                label_group = label_name_to_group.get(label_name, None)
                if label_group is not None and label_group in occupied_groups:
```

Reply:
I updated to consider about other `GroupType`s